ratis-server/src/main/java/org/apache/ratis/server/storage/SnapshotManager.java

```diff
@@ -63,7 +64,7 @@ public class SnapshotManager {
   private final Supplier<File> snapshotDir;
   private final Supplier<File> snapshotTmpDir;
   private final Function<FileChunkProto, String> getRelativePath;
-  private final Supplier<MessageDigest> digester = JavaUtils.memoize(MD5Hash::getDigester);
+  private MessageDigest digester;
```

Comment:
Let's move it to a local variable?

Reply:
I explained it in the code snippet you provided, please take a look.